hardhat.config.js

```diff
@@ -69,11 +69,25 @@ task("upgrade-facets", "Upgrade existing facets, add new facets or remove existi
   .addParam("newVersion", "The version of the protocol to upgrade to")
   .addParam("env", "The deployment environment")
   .addParam("functionNamesToSelector", "JSON list of function names to selectors")
-  .addOptionalParam("facetConfig", "JSON list of facets to upgrade")
```

Comment:
I don't think removing `facetConfig` from the parameters is ok, since there are some scripts that are passing in this parameter (scripts in `scripts/migrations/` and `/home/klemen/boson/boson-protocol-contracts/test/util/upgrade.js`, which use hardhat tasks to invoke the method). Keeping it optional should not break anything.